modules/ztp-deploying-a-site.adoc

```diff
@@ -78,6 +78,12 @@ include::snippets/ztp_example-sno.yaml[]
 For more information about BMC addressing, see the "Additional resources" section. The `installConfigOverrides` and  `ignitionConfigOverride` fields are expanded in the example for ease of readability.   
 ====
 
++
+[NOTE]
+====
+To override the default `BareMetalHost` CR for a node, you can reference the override CR in the `spec.clusters.clusterName.nodes/<node-id>/crTemplates.BareMetalHost` field. Ensure that you set the `argocd.argoproj.io/sync-wave: "3"` annotation in your override `BareMetalHost` CR.
```

Comment:
```suggestion
To override the default `BareMetalHost` CR for a node, you can reference the override CR in the `crTemplates` field under node level in the SiteConfig. Ensure that you set the `argocd.argoproj.io/sync-wave: "3"` annotation in your override `BareMetalHost` CR.
```

Reply:
Rephrase as you see it fits, since we are not describing API, we must refrain from providing node id

Reply:
> To override the default `BareMetalHost` CR for a node, you can reference the override CR in the node-level `crTemplates` field in the `SiteConfig` CR. Ensure that you set the `argocd.argoproj.io/sync-wave: "3"` annotation in your override `BareMetalHost` CR.